content/how-to/hot-module-reload.md

```diff
@@ -171,12 +171,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
 
-import App from './components/App';
-
 const render = () => {
   ReactDOM.render(
     <AppContainer>
-      <App/>
+      { require('./components/App').default }
```

Comment:
I think the code will be more clear if we did not inline it:

```js
  const App = require('./components/App').default
  ReactDOM.render(
    <AppContainer>
      <App />
```

Also, `require('./components/App').default` results in a React Component instead of a React Element, so I believe it doesn’t work unless you change it to this:

```js
      {React.createElement(require('./components/App').default)}
```